src/Space.cpp

```diff
@@ -149,6 +151,28 @@ Space::Space(Game *game, RefCountedPtr<Galaxy> galaxy, const Json &jsonObj, doub
 	for (Body *b : m_bodies)
 		b->PostLoadFixup(this);
 
+	// some spaceports could be moved, now their physical bodies were loaded from
+	// json, with offsets, now we should move the system bodies also so that
+	// there is no mismatch
+	for (auto sbody : m_starSystem->GetBodies())
```

Comment:
Style point but please use brackets on multi-line for loops

Reply:
Done!